lib/msf/core/post/windows/process.rb

```diff
@@ -44,20 +44,23 @@ def get_notepad_pathname(bits, windir, client_arch)
   end
 
   #
-  # Injects a reflective DLL in to a process, and executes it.
+  # Injects a reflective DLL into a process, and executes it.
   #
   # @param rdll_path [String] The path to the DLL to inject
   # @param param     [String, Integer, nil] The parameter to pass to the DLL's entry point. If this value is a String
   #   then it will first be written into the process memory and then passed by reference. If the value is an Integer,
   #   then the value will be passed as is. If the value is nil, it'll be passed as a NULL pointer.
-  # @param pid       [Integer] The process ID to inject to, if unspecified, a new instance of notepad.exe will be
-  #   launched to host the injected DLL.
+  # @param pid       [Integer] The process ID to inject to, if unspecified, a new instance of a random EXE from the
+  #   process_list array will be launched to host the injected DLL.
   def execute_dll(rdll_path, param=nil, pid=nil)
+    process_list = ['sigverif', 'netsh', 'nslookup', 'winver', 'nbtstat -r 200', 'dxpserver', 'sndvol', 'netstat 200']
     if pid.nil?
-      print_status('Launching notepad to host the DLL...')
-      notepad_process = client.sys.process.execute('notepad.exe', nil, { 'Hidden' => true })
+      offset_num = rand(0..process_list.length-1)
+      process_cmd = process_list[offset_num]
```

Comment:
You can use the builtin [`Array#sample`](https://www.rubydoc.info/stdlib/core/Array:sample) method here.

```suggestion
    process_cmd = process_list.sample
```

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/15384/commits/e5a91736fd2e8dd00948d7df2dcfe9fdbc1c57cb